Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -1152,3 +1152,9 @@ zurueckZuArbeitsschritt=Zur\u00FCck zu Aufgabe
 zurueckZurStartseite=Zur\u00FCck zur Startseite
 zusaetzlicheDetails=Zus\u00E4tzliche Details
 zusammenfassung=Zusammenfassung
+neu=Neu
```

Comment:
Why you are using a German message key? New keys should be in English.

Reply:
Because using "new" as a key is not possible that's can be interpreted as the java new Object().
that's why also I have modified it to newElement.